modules/reference/pages/cluster-properties.adoc

```diff
@@ -2142,26 +2137,25 @@ If set to `0`, no minimum is enforced.
 
 *Default:* `256`
 
+*Related topics*:
```

Comment:
@Deflaimun not sure if this was already raised, but could we standardize where the related topics appear? I think preferably it would be after all the other bullet points that are pertinent to that property